client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -30,17 +32,16 @@ class DomainRow extends PureComponent {
 		currentRoute: PropTypes.string,
 		disabled: PropTypes.bool,
 		domain: PropTypes.object.isRequired,
-		domainDetails: PropTypes.object,
```

Comment:
Getting rid of this one as I'm merging all the data in the DomainsTable component